### ISSUE TITLE
[PARAMETERS] Increase the resolution

### ISSUE THREAD

**Issue by FelixDubois** [privatenumber/snap-tweet#39]
It might be interesting to add a parameter to increase the resolution of the snaps. Right now the snaps have a small resolution which makes them blurred.

**Comment by Jolly-Pirate** [comment]
The scaling https://github.com/privatenumber/snap-tweet/pull/38 did the job of increasing the resolution, didn't it? It looked good to me.

**Comment by FelixDubois** [comment]
Yes but if I understood well, to propose a new commit first I have to create an issue

**Comment by privatenumber** [comment]
As demonstrated in the README, the screenshots I get are high resolution:
<img src="https://raw.githubusercontent.com/privatenumber/snap-tweet/develop/.github/example.png">

Can you describe your problem in more detail?
